RInAppMessaging/Classes/Extensions/UIApplication+IAM.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+extension UIApplication {
```

Comment:
Are the below extension functions testable?

Reply:
I don;t see a way to write a test that make sense.

Reply:
Does the unit test target have a host app? If so, you could check that these functions return a non-nil value. I concede that doesn't test much though.